modules/helpers/crew.js

```diff
@@ -79,19 +88,21 @@ export function deregister_crew(vehicle_id, crew_member, crew_role) {
 
 /**
  *
- * @param vehicle_id actor ID of the vehicle receiving the crew change
+ * @param vehicle_actor actor object of the vehicle receiving the crew change
  * @param crew_member actor ID of the crew being changed
  * @param old_crew_role old role of the crew member
  * @param new_crew_role new role of the crew member
  */
-export function change_role(vehicle_id, crew_member, old_crew_role, new_crew_role) {
-    CONFIG.logger.debug("Got role change request: ", vehicle_id, crew_member, old_crew_role, new_crew_role);
-    const vehicle_actor = game.actors.get(vehicle_id);
-    const flag_data = vehicle_actor.getFlag('starwarsffg', 'crew');
+export async function change_role(vehicle_actor, crew_member, old_crew_role, new_crew_role) {
+    CONFIG.logger.debug(
+      `Got role change request: vehicle ID: ${vehicle_actor} | crew ID: ${crew_member} | old role: ${old_crew_role} | new role: ${new_crew_role}`
+    );
+    const flag_data = await vehicle_actor.getFlag('starwarsffg', 'crew');
```

Comment:
Harmless nit: don't think `Document#getFlag` is async.

https://foundryvtt.com/api/classes/foundry.abstract.Document.html#getFlag